src/components/tables/TableActionsDropdown.tsx

```diff
@@ -0,0 +1,65 @@
+import { DeleteOutlined, MoreOutlined } from '@ant-design/icons';
+import { Button, Dropdown, MenuProps } from 'antd';
+import { useMemo, useState } from 'react';
+import { DeleteTableModal } from '../modals/DeleteTableModal';
+
+interface TableActionDropdownProps {
+  catalog: string;
+  schema: string;
+  table: string;
+  tableFullName: string;
+}
+
+enum TableActionsEnum {
+  Delete,
+  Rename
+}
+
+
+export default function TableActionsDropdown({
+  catalog,
+  schema,
+  table,
+  tableFullName,
+}: TableActionDropdownProps) {
+  const [dropdownVisible, setDropdownVisible] = useState<boolean>(false);
+  const [action, setAction] = useState<TableActionsEnum | null>(null);
+
+  const menuItems = useMemo(
+    (): MenuProps['items'] => [
+      {
+        key: 'deleteTable',
+        label: 'Delete Table',
+        onClick: () => setAction(TableActionsEnum.Delete),
+        icon: <DeleteOutlined />,
+        danger: true
+      },
+    ],
+    []
+  );
+
+  return (
+    <>
+      <Dropdown
+        menu={{ items: menuItems }}
+        trigger={['click']}
+        onOpenChange={() => setDropdownVisible(!dropdownVisible)}>
+        <Button
+          type="primary"
+          shape='circle'
```

Comment:
Can we make it `type="text` and squared?
as this `...` dropdown will also show up in other pages like Catalog or Schema details, and it will have primary button next to it

Reply:
Something like this @yc-shawn ?
![image](https://github.com/user-attachments/assets/1934696e-128f-4864-82a8-3458f4d3558f)

Reply:
This is my code:

`return (
    <>
      <Dropdown menu={{ items: menuItems }} trigger={['click']}>
        <Button type="primary">
          <MoreOutlined />
        </Button>
      </Dropdown>

      {/* Render the EditCatalogModal */}
      {openModal === CreateItemsEnum.Edit && (
        <EditCatalogModal open={true} closeModal={closeModal} />
      )}
    </>
  );`

Reply:
similar but `type="text"` instead of `type="primary"`